server/test/unit/com/thoughtworks/go/server/security/LdapUserSearchTest.java

```diff
@@ -102,8 +103,8 @@ public void shouldThrowUserNameNotFoundExceptionWhenNoUserFound_WithMultipleSear
         try {
             spy.searchForUser("username");
             fail("Should have thrown up");
-        } catch (UsernameNotFoundException e) {
-            assertThat(e.getMessage(), is("User username not found in directory."));
+        } catch (BadCredentialsException e) {
+            assertThat(e.getMessage(), is("Bad credentials"));
         }
```

Comment:
We must start using http://junit.org/apidocs/org/junit/rules/ExpectedException.html
For asserting exceptions.